src/makeInfinityAdder.js

```diff
@@ -4,7 +4,27 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(num) {
+    if (num === undefined) {
```

Comment:
Consider using 'num == null' instead of 'num === undefined' to also handle null values, which might be a more robust check for undefined or null input.